enterprise/backend/src/metabase_enterprise/enhancements/models/native_query_snippet/permissions.clj

```diff
@@ -32,7 +32,7 @@
 (defenterprise can-create?
   "Can the current User save a new Snippet with the values in `m`?"
   :feature :any
-  [model m]
+  [_model m]
```

Comment:
This is a Kondo error that is caught by the custom `defenterprise` hook that was not caught before.